package-lock.json

```diff
@@ -5342,9 +5342,9 @@
       }
```

Comment:
Shouldn't you run an `npm install` to ensure the update of the actual package though?